recipes-dom0/dom0-image-thin-initramfs/files/meta-xt-prod-extra/recipes-bsp/u-boot/u-boot_%.bbappend

```diff
@@ -5,32 +5,15 @@ UBOOT_MACHINE ?= "xenguest_arm64_defconfig"
 
 SRCREV = "${AUTOREV}"
 SRC_URI = "\
-    git://github.com/arminn/u-boot.git;protocol=https;branch=mr_rcar_pvblock; \
+  ${@bb.utils.contains('XT_GUESTS_INSTALL','doma','git://github.com/arminn/u-boot.git;protocol=https;branch=mr_rcar_pvblock;','${SRC_URI}',d)}; \
 "
 FILES_${PN} += " \
-      /xt/doma \
+      ${@bb.utils.contains('XT_GUESTS_INSTALL', 'doma', '/xt/doma', '', d)} \
```

Comment:
I think we shouldn't use direct "/xt" as we have a variable for that: XT_DIR_ABS_SHARED_BOOT_DOMA

Reply:
now, depends on install_domain
GUEST = "${@bb.utils.contains('XT_GUESTS_INSTALL', 'doma', 'doma', 'domu', d)}"

FILES_${PN} += " \
   /xt/${GUEST} \
"